ansible/roles/jenkins-worker/templates/systemd.service.j2

```diff
@@ -23,7 +23,7 @@ Environment="NODE_COMMON_PIPE=/home/{{ server_user }}/test.pipe"
 Environment="NODE_TEST_DIR=/home/{{ server_user }}/tmp"
 Environment="OSTYPE=linux-gnu"
 
-ExecStart=/usr/bin/java -Xmx{{ server_ram|default('128m') }} \
+ExecStart={{ java_path[os] }} -Xmx{{ server_ram|default('128m') }} \
```

Comment:
This causes some trouble. ExecStart *must* be an absolute path, or systemd refuses to use it. The path is usually /usr/bin, but on centos-ppc64le, its in /hom/iojs/.../bin/java (see comment below). So, I use the `java_path`, but its hard to know if that will work for everyone.

Is this the right way?

Reply:
I think it may be the right way but will likely break other platforms as I'm not sure that java_path is fully populated for all of the existing platforms.

Reply:
Maybe `{{ java_path[os] || '/usr/bin/java' }}` ?

Reply:
I backed this out, an instead I untar java to /opt, and symlink /usr/bin/java (the expeted path) to /opt/BLAH/bin/java. It worked with -1, according to `systemctl status jenkins`, so now I'm reansibling -1 and -2.